functions/memorystore/redis/visitcount.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_memorystore_redis]
+
+// Package visitcount provides a Cloud Function that connects
+// to a managed Redis instance.
+package visitcount
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gomodule/redigo/redis"
+)
+
+var redisPool *redis.Pool
+
+// Initialize the connection pool on instance startup
+func init() {
+	redisHost := os.Getenv("REDISHOST")
+	if redisHost == "" {
+		log.Fatal("REDISHOST must be set")
+	}
+	redisPort := os.Getenv("REDISPORT")
+	if redisPort == "" {
+		log.Fatal("REDISPORT must be set")
+	}
+	redisAddr := fmt.Sprintf("%s:%s", redisHost, redisPort)
+
+	const maxConnections = 10
+	redisPool = &redis.Pool{
+		MaxIdle: maxConnections,
+		Dial: func() (redis.Conn, error) {
+			c, err := redis.Dial("tcp", redisAddr)
+			if err != nil {
+				log.Fatal("Error connecting to Redis instance")
```

Comment:
```suggestion
				log.Fatal("redis.Dial: %v", err)
```

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#return-errors

Reply:
Done